content/tutorials/single-page-tutorials/send-shimmer-tokens-with-javascript.md

```diff
@@ -0,0 +1,265 @@
+```
+
+Next you need to install the [iota client library](https://github.com/iotaledger/iota.rs/tree/develop/bindings/nodejs). Be sure, you install the `@next` version to use the Shimmer network. 
+```bash
+npm i @iota/client@3.0.0-alpha.6
+```
+
+
+## 1. Get your first address.
+
+Next we want to generate an address, where we want to deposit some Shimmer testnet Tokens.
+
+Just create a new file called `get_address.js` in a `scripts` directory and add the content below. On Linux based machines, you can use the command `touch` to create a new file.
+
+```bash!
+mkdir scripts
+touch scripts/get_address.js
+```
+
+Content:
+
+```javascript
+
+const {
+  Client,
+  CoinType,
+  initLogger,
+  SHIMMER_TESTNET_BECH32_HRP,
+} = require("@iota/client");
+
+
+async function run() {
+
+  const client = new Client({
+    // Insert your node URL in the .env.
+    nodes: ['https://api.testnet.shimmer.network'],
+  });
+
+  // You should only generate mnemonics for development or testing
+  const mnemonic = await client.generateMnemonic();
+  console.log('Mnemonic: ' + mnemonic);
+
+  const secretManager = {
+    Mnemonic: mnemonic,
+  };
+
+  // Generate addresses by providing all inputs, that way it can also be done offline without a node.
+  const offlineGeneratedAddresses = await client.generateAddresses(
+    secretManager,
+    {
+      coinType: CoinType.Shimmer,
+      accountIndex: 0,
+      range: {
+        start: 0,
+        end: 1,
+      },
+      internal: false,
+      // Generating addresses with client.generateAddresses(secretManager, {}), will by default get the bech32_hrp (Bech32
+      // human readable part) from the nodeinfo, generating it "offline" requires setting it in the generateAddressesOptions
+      bech32Hrp: SHIMMER_TESTNET_BECH32_HRP,
+    }
+  );
+  console.log(
+    "Generated public address:",
+    offlineGeneratedAddresses[0]
+  );
+}
+
+run().then(() => process.exit());
+
+```
+
+If we run this script, you should see your Shimmer address in your console output.
+
+```bash
+node scripts/get_address.js
+```
+
+This prints something like this:
+
+> Mnemonic: deputy cousin oxygen quiz also odor clever candy borrow know junk method logic alert give history toy dolphin enact shift tooth wreck verify evil
+Generated public address: rms1qpzkrdmt0dja3sch2mt2sapj46j4ywatxlxrgneynqtcp8ltyzf7yz75ser
+
+Save your Mnemonic in a file, we need this to send your tokens in the last step.
+
+:::tip
+
+In testnet addresses begin with `rms...`, whereas in the Mainnet addresses will begin with `smr...`.
+
+:::
+
+## 2. Get some testnet Tokens.
+
+Now we need to visit the Faucet to get some tokens. Just input your address on the website and you will get some testnet tokens!
+
+> Shimmer testnet faucet: https://faucet.testnet.shimmer.network/
+
+Did it work? Let's check our address balance!
+
+## 3. Read your balance
+
+Now, let's create a new file called `get_balance.js` and add the code below!
+
+```bash
+touch scripts/get_balance.js
+```
```

Comment:
I don't think this is necessary, they'll need to edit the file with their text editor of choice anyhow.